tailwind.config.js

```diff
@@ -52,7 +52,15 @@ export default {
 			anakiwa: '#b0cdff',
 			beer: '#f7931a',
 			fulvous: '#de7900',
-			water: '#d1e3ff'
+			water: '#d1e3ff',
+			info: {
```

Comment:
as discussed offline, let's keep it as primitive colors for now, and work on the mapping separately